tutor/interactive.py

```diff
@@ -145,6 +145,40 @@ def ask_questions(config: Config, run_for_prod: Optional[bool] = None) -> None:
             config,
             defaults,
         )
+    if plugins.is_loaded("surveyreport"):
```

Comment:
This PR in the current form is adding to the tutor core a custom way of asking questions that only apply to the survey report plugin. I don't think that would be the way of doing this feature.
Tutor is build on plugability, we need to create a new filter right here that we can later extend from the survey report.